.github/workflows/unit-tests.yaml

```
@@ -20,10 +20,24 @@ jobs:
           - ansible: stable-2.16
             python: "3.10"
     steps:
-      - name: Perform unit testing
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)